syntax(done and to do).md

```diff
@@ -0,0 +1,34 @@
+## Included Syntax
+<ul> 
+<li>  Paragraph 
+<li>  New line
+<li> Horizontal line
+<li> Comments
+<li> Simple text formatting tags
+<li> Headings
+<li> Images
+<li> Links
+<li> Lists
+<li> tables
+
+## TO DO:
```

Comment:
Let's get these into issues so anyone can pick them up and help add to the tutorial!